examples/geojson-animation/webpack.config.js

```diff
@@ -5,6 +5,16 @@
 const resolve = require('path').resolve;
 const webpack = require('webpack');
 
+const BABEL_CONFIG = {
```

Comment:
But why just for this example. Why was it not necessary for any of the others? 

How is the webpack.config of this example even involved when we build the website?